password/impl/src/main/java/org/wildfly/security/password/impl/ElytronMessages.java

```diff
@@ -43,7 +44,7 @@
 @ValidIdRanges({
     @ValidIdRange(min = 2, max = 4),
     @ValidIdRange(min = 5151, max = 5151),
-    @ValidIdRange(min = 8013, max = 8028)
+    @ValidIdRange(min = 8013, max = 8030)
```

Comment:
Thanks for the updates @lvydra! Just one more small change needed here. When adding new messages where the current valid range doesn't have remaining IDs, we need to allocate a new range in https://github.com/wildfly-security/wildfly-elytron/blob/1.x/ELY_Messages.txt to make sure we don't overlap with other IDs.

So